### ISSUE TITLE
Feature Request: Text clipping workaround for .renderViewToPDF()

### ISSUE THREAD

**Issue by davidsara** [archimatetool/archi-scripting-plugin#130]
Hello!
Is it possible to add Text clipping workaround as a parameter of **.renderViewToPDF()** function?
I'm using jArchi script for batch PDF export and need this parameter to fix issues with missing part of last character of some texts.

![image](https://github.com/archimatetool/archi-scripting-plugin/assets/20066733/341d8ab8-a1c1-4dcd-8fb1-fb57844a4a22)

Thanks!
David

**Comment by Phillipus** [comment]
This can happen with certain fonts. In the meantime you could try a different font. I'll add this one to the list.

**Comment by davidsara** [comment]
Thank you!

**Comment by Phillipus** [comment]
For the next version of jArchi we will support a map of options for the SVG and PDF image export.

The current [API](https://github.com/archimatetool/archi-scripting-plugin/wiki/Model-Functions) for SVG and PDF is:

`$.model.renderViewAsSVGString(view, setViewBox)`
`$.model.renderViewToSVG(view, filePath, setViewBox)`
`$.model.renderViewToPDF(view, filePath)`

We can add extra functions that take a map of options in the arguments:

`$.model.renderViewAsSVGString(view, options)`
`$.model.renderViewToSVG(view, filePath, options)`
`$.model.renderViewToPDF(view, filePath, options)`

Examples:

```js
var svgOptions = {setViewBox: true, viewBoxBounds: "0 0 1000 200", textAsShapes: false, embedFonts: true, textOffsetWorkaround: true};
var svgString = $.model.renderViewAsSVGString(view, svgOptions);
$.model.renderViewToSVG(view, "pathTo/test.svg", svgOptions);

var pdfOptions = {textAsShapes: false, embedFonts: true, textOffsetWorkaround: true};
$.model.renderViewToPDF(view, "pathTo/test.pdf", pdfOptions);
```
Each option can be omitted if required so you can set just one or two options. The defaults are:

```js
setViewBox: true (this option applies only to SVG export)
viewBoxBounds: null (this option applies only to SVG export)
textAsShapes: true
embedFonts: false
textOffsetWorkaround: false
```

**Comment by Phillipus** [comment]
This is implemented in jArchi 1.7.
